src/NHSD.BuyingCatalogue.Organisations.Api/Controllers/OdsController.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using NHSD.BuyingCatalogue.Organisations.Api.Models;
+using NHSD.BuyingCatalogue.Organisations.Api.ViewModels.Organisations;
+
+namespace NHSD.BuyingCatalogue.Organisations.Api.Controllers
+{
+    [Authorize(Policy = Policy.CanAccessOrganisation)]
+    [Route("api/v1/ods")]
+    [ApiController]
+    [Produces("application/json")]
+    public sealed class OdsController : Controller
+    {
+        [HttpGet]
+        [Route("{odsCode}")]
+        public ActionResult GetByOdsCodeAsync(string odsCode)
+        {
+            if (odsCode is null)
+                throw new ArgumentNullException(nameof(odsCode));
+
+            if (string.IsNullOrWhiteSpace(odsCode))
+                return NotFound();
+
+            // Canned data
+            return Ok(new OdsViewModel
+            {
+                OrganisationId = Guid.NewGuid(),
```

Comment:
I don't think we need the organisationId as for this endpoint we are treating the ODS code as the unique ID.

Reply:
That is right, had a look at the updated JSON contract and we're not returning the org id. I'll remove it from the OdsViewModel. 
Good spot 👍